### PR TITLE
[imporve](flex) support scientific notation(aEb) parser

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes

Issue Number: close #https://github.com/apache/doris/issues/22189

aEb

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.5 seconds
 stream load tsv:          508 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.2 seconds inserted 10000000 Rows, about 342K ops/s
 storage size: 17162543252 Bytes

File: fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java [hunk 1]
```diff
@@ -121,7 +121,7 @@ public static void execUpdate(String template, Map<String, String> params) throw
     }
 
     public static List<ResultRow> execStatisticQuery(String sql) {
-        if (FeConstants.disableInternalSchemaDb) {
```
Comment (zhangstar333) [inline fe/fe-core/src/main/java/org/apache/doris/statistics/util/StatisticsUtil.java:124]: https://github.com/apache/doris/pull/22261, only used for compile P0 successed

**Comment by zhangstar333** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.35 seconds
 stream load tsv:          507 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17169757300 Bytes

**Comment by HappenLee** [comment]
Add regression test please
like “-1e9” “0e0”

**Comment by zhangstar333** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**